nifi-nar-bundles/nifi-azure-bundle/nifi-azure-processors/src/main/java/org/apache/nifi/processors/azure/storage/PutAzureDataLakeStorage.java

```diff
@@ -83,11 +88,23 @@ public class PutAzureDataLakeStorage extends AbstractAzureDataLakeStorageProcess
             .allowableValues(FAIL_RESOLUTION, REPLACE_RESOLUTION, IGNORE_RESOLUTION)
             .build();
 
+    public static final PropertyDescriptor BASE_TEMPORARY_PATH = new PropertyDescriptor.Builder()
+            .name("base-temporary-path")
+            .displayName("Base Temporary Path")
+            .description("The Path where the temporary directory will be created. The Path name cannot contain a leading '/'." +
+                    " The root directory can be designated by the empty string value. Non-existing directories will be created." +
+                    "The Temporary File Directory name is " + TEMP_FILE_DIRECTORY)
+            .defaultValue("")
+            .expressionLanguageSupported(ExpressionLanguageScope.FLOWFILE_ATTRIBUTES)
+            .addValidator(new DirectoryValidator("Temp File Directory"))
```

Comment:
Should this label be adjusted to match the property name?
```suggestion
            .addValidator(new DirectoryValidator("Base Temporary Path"))
```